site/public/css/server.css

```diff
@@ -1619,15 +1576,6 @@ end of styles used in the admin gradeable page
     border: 1px solid var(--text-black);
 }
 
-.wrapper_layout_header > th {
-    padding: 4px;
-    border: 1px solid var(--text-black);
-}
-
-.wrapper-upload .wrapper_layout_header {
-    background-color: var(--standard-pale-gray);
-}
-
```

Comment:
I think this class is still in use in `site/app/templates/admin/UploadWrapperForm.twig`, so perhaps this should be 
```
.wrapper-layout-header {
    color: var(--text-black);
}
```
so that the table works on dark mode. (of course, change the class to use dashes as well)
(Sorry, GitHub doesn't let me insert a suggestion on deleted lines)